wp-content/themes/goonj-crm/functions.php

```diff
@@ -381,8 +393,12 @@ function goonj_handle_user_identification_form() {
 		->setLimit( 1 )
 		->execute();
 
+		// Recent camp data
+		$recentCamp = $collectionCampResult->first() ?? null;
+		$display_name = $found_contacts['display_name'];
+
 		if ( $purpose === 'dropping-center' ) {
-			wp_redirect( get_home_url() . '/dropping-center/intent/#?Collection_Camp_Core_Details.Contact_Id=' . $found_contacts['id'] );
+			wp_redirect( get_home_url() . '/dropping-center/intent/#?Collection_Camp_Core_Details.Contact_Id=' . $found_contacts['id'] . '&Dropping_Centre.Name=' . $display_name . '&Dropping_Centre.Contact_Number=' . $phone);
 			exit;
```

Comment:
Here, we need to prefill the dropping center intent details with the volunteer's name and contact information.